packages/turf-transform-scale/index.js

```diff
@@ -30,26 +30,19 @@ var getCoords = invariant.getCoords;
  * scaledPoly.properties = {stroke: '#F00', 'stroke-width': 4};
  * var addToMap = [poly, scaledPoly];
  */
-module.exports = function (geojson, factor, fromCenter, mutate) {
+module.exports = function (geojson, factor, fromCorner, mutate) {
     // Input validation
     if (!geojson) throw new Error('geojson is required');
     if (typeof factor !== 'number' && factor === 0) throw new Error('invalid factor');
     if (!factor) throw new Error('factor is required');
 
     // Default params
-    var isMutate = mutate === false || mutate === undefined;
-    var isCenter = fromCenter === true || fromCenter === undefined;
-    var origin;
+    var isMutate = (mutate === false || mutate === undefined);
+    var isPoint = (geojson.type === 'Point' || geojson.geometry && geojson.geometry.type === 'Point');
+    var origin = defineOrigin(geojson, fromCorner);
 
     // Shortcut no-scaling
-    if (factor === 1) return geojson;
-
-    // Define origin
-    if (isCenter) origin = centroid(geojson);
-    else {
-        var box = (geojson.bbox) ? geojson.bbox : bbox(geojson); // [ west, south, east, north ]
-        origin = point([box[0], box[1]]);
-    }
+    if (factor === 1 || isPoint) return geojson;
 
     // Clone geojson to avoid side effects
     if (isMutate) geojson = JSON.parse(JSON.stringify(geojson));
```

Comment:
@DenisCarriere I'd put this before any `return`, since the `mutate` behaviour should be respected even if there is no actual `scaling`. That's why my original [multiple `mutate === undefined`](https://github.com/Turfjs/turf/pull/759#discussion_r118824374)